plugins/woocommerce/src/Utilities/I18nUtil.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * A class of utilities for dealing with internationalization.
+ */
+
+namespace Automattic\WooCommerce\Utilities;
+
+/**
+ * A class of utilities for dealing with internationalization.
+ */
+final class I18nUtil {
+	/**
+	 * A cache for the i18n units data.
+	 *
+	 * @var array $units
+	 */
+	private static $units;
+
+	/**
+	 * Get the translated label for a weight unit of measure.
+	 *
+	 * This will return the original input string if it isn't found in the units array. This way a custom unit of
+	 * measure can be used even if its' not getting translated.
```

Comment:
Final minor nit, sorry!

```suggestion
	 * measure can be used even if it's not getting translated.
```